gtsam/hybrid/HybridGaussianFactorGraph.h

```diff
@@ -259,16 +314,44 @@ class GTSAM_EXPORT HybridGaussianFactorGraph
       const Eliminate& function = EliminationTraitsType::DefaultEliminate,
       OptionalVariableIndex variableIndex = boost::none) const;
 
+  /// Sequential elimination overload for hybrid
   boost::shared_ptr<BayesNetType> eliminateSequential(
       OptionalOrderingType orderingType = boost::none,
       const Eliminate& function = EliminationTraitsType::DefaultEliminate,
       OptionalVariableIndex variableIndex = boost::none) const;
 
+  /// Sequential elimination overload for hybrid
   boost::shared_ptr<BayesNetType> eliminateSequential(
       const Ordering& ordering,
       const Eliminate& function = EliminationTraitsType::DefaultEliminate,
       OptionalVariableIndex variableIndex = boost::none) const;
 
+  /**
+   * @brief Custom elimination function which computes the correct
```

Comment:
OK, this is my biggest beef. More in the final review comment.

And add override to all functions you are overriding.

Reply:
The problem I am trying to tackle is computing the correct `q(\mu)` terms for each mode sequence. This requires having all the continuous conditionals as a BayesNet that we can use.

Unfortunately, the elimination method in `EliminateableFactorGraph` calls `DefaultEliminate` (a static method) with one key at a time. By the time we get to the last continuous variable to eliminate, we only have access to the one conditional and not all the conditionals needed to make the BayesNet. Since `DefaultEliminate` is a static method, we also can't do something like `this->conditionals()`.

Hence the need to override, so we can eliminate all the continuous variables first to get the continuous BayesNet and get the `P(X | M, Z)` posterior as well as the terms in the `P(M | Z)` posterior (e.g. the `q(\mu)` term and the determinant term). This way we can perform all these distinct operations together in sequence when a user calls `HybridFactorGraph::eliminateSequential` or `HybridFactorGraph::eliminateMultifrontal`.

If you have a better way of doing this that doesn't involve overriding, I am all ears. 🙂

Reply:
Also, I can't mark these methods as override since the base class functions are not virtual. I can update that on `EliminateableFactorGraph` but wanted to let you know first.